### ISSUE TITLE
Create tutorial for google colab

### ISSUE THREAD

**Issue by textpotential** [hlab-repo/hlab-embedded#6]
Simple tutorial for humanities people on why and how to use google colab, maybe with a few sample experiments.

**Comment by timothybeal** [comment]
Awesome! Keep sample experiments super kinder simple! :)

**Comment by textpotential** [comment]
Totally! Things like Hello World!

**Comment by textpotential** [comment]
@timothybeal might need to touch base with you a bit to make sure I know details of case.edu google account stuff.

**Comment by textpotential** [comment]
I should actually move this issue to the hlab repo, not here. Will see if I can move it or will just close and recreate.

**Comment by timothybeal** [comment]
Yes happy to do that! Also think abt our best way to feature these under the Learning area on the main site which should provide SEO help.

**Comment by textpotential** [comment]
@timothybeal you have a generic hlab repo that runs the github pages site, right? I don't think I have access to that repo.

**Comment by timothybeal** [comment]
This is working for me bw the app and email and leaving git on browser open btw so thanks!

**Comment by timothybeal** [comment]
I'll make sure you have access (thot you did)

**Comment by timothybeal** [comment]
Actually that site is under timothybeal not hlab-repo ... Need for organizing is emerging here. We should at least add an hlab-repo/learning parallel to the two projects. Will do that now before start interviews
